build-system/compile/bundles.config.js

```diff
@@ -44,8 +46,8 @@ exports.jsBundles = {
   'alp.max.js': {
     srcDir: './ads/alp/',
     srcFilename: 'install-alp.js',
-    destDir: './dist',
-    minifiedDestDir: './dist',
+    destDir: `${outputDir}`,
```

Comment:
```suggestion
    destDir: outputDir,
```

everywhere, no need to use the string interpolation

Reply:
Oops! Fixed now.